applications/GeoMechanicsApplication/custom_constitutive/incremental_linear_elastic_interface_law.h

```diff
@@ -42,6 +54,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoIncrementalLinearElasticInterface
                const ProcessInfo&  rCurrentProcessInfo) const override;
 
 private:
+    GeoIncrementalLinearElasticInterfaceLaw() : mConstitutiveLawDimension(nullptr) {}
```

Comment:
A default-constructed `std::unique_ptr` will guard a `nullptr`, so I think the following will do:
```suggestion
    GeoIncrementalLinearElasticInterfaceLaw() = default;
```

Reply:
done